python/tvm/expr.py

```diff
@@ -14,36 +14,51 @@
   assert(isinstance(y, tvm.expr.Add))
   assert(y.a == x)
 """
-# pylint: disable=missing-docstring
+# pylint: disable=missing-docstring, unnecessary-lambda
 from __future__ import absolute_import as _abs
 from ._ffi.node import NodeBase, NodeGeneric, register_node
 from . import make as _make
+from . import generic as _generic
 from . import _api_internal
 
+__op_priority__ = 1
+
+def _bind_generic_ops():
```

Comment:
why do we need bind here? insted of put implementation directly into generic?

Reply:
Well, I was trying to separate interface and implementation into different modules. generic.py defines interface only, and expr.py defines all implementation details (e.g., _make.Add(), _make.Sub()) and bind whenever appropriate.

Anyway, I will put those implementation directly in generic.py as you suggested.